### DISCUSSION TITLE
Possible to create a bounding border to ScatterPlot Layer data?

### DISCUSSION THREAD

**Discussion by btbritz** [visgl/deck.gl#7844] [Q&A]
The idea is to build a sort of bounding border/background around different chunks of data. Not sure if there was built in functionality to get something similar to this in the ScatterPlot Layer:

<img width="997" alt="Screen Shot 2023-04-18 at 10 20 52 AM" src="https://user-images.githubusercontent.com/3333099/232855749-45d9bee1-8abf-4b46-ba9a-f93c630a1fd4.png">

If so, where can I look for more information?

If not, Is it possible to create something like this using the different extensions and other functionality Deck.gl provides? Any help with an implementation plan would be awesome!

Thanks!

**Comment by lianzhao** [comment]
try [turf/concave](https://turfjs.org/docs/#concave)?
